### PR TITLE
(PLATFORM-4038) Add new devbox Dockerfile

### PR DESCRIPTION
**Pull request by Grunny**
## Links

* http://wikia-inc.atlassian.net/browse/PLATFORM-4038

## Description

Add new devbox Dockerfile as a draft base for our new LTS devboxes.

## Reviewers

@Wikia/core-platform-team

### CODE REVIEW

File: config/fastboot-server.js [hunk 1]
```diff
@@ -12,10 +12,20 @@ module.exports = (function () {
 
   if (process.env.WIKIA_ENVIRONMENT === 'dev') {
     const devDomain = (process.env.WIKIA_DATACENTER === 'poz') ? 'pl' : 'us';
+    if (process.env.MOBILE_WIKI_LOG_TCP_PORT) {
```
Comment (abador) [inline config/fastboot-server.js]: Maybe we should check both port and host? Just to be sure if nobody overwrites it.

Comment (harnash) [reply config/fastboot-server.js]: Fix in 164a8fa59

File: README.md [hunk 2]
```diff
@@ -90,6 +90,42 @@ If you're in the #iris-tech channel, you can also make use of `sandbot` which is
...
+  echo LOCAL_USER_ID=$(id -u $USER) > .env
+	```
+
+You will need a Github token which will be used to fetch packages from private repo.
+To get a token go to [your Github settings](https://github.com/settings/tokens) and generate new token
+or use one you have generated earlier. I recommend to store it somewhere safe and just save it per your
+ssh session:
+
+```bash
+export GITHUB_TOKEB=<YOUR_TOKEN_HERE>
```
Comment (abador) [inline README.md]: typo

File: Dockerfile.devbox [hunk 3]
```diff
@@ -0,0 +1,23 @@
+FROM node:8.9.1-alpine
+
+WORKDIR /app
+
+# mount app source
+VOLUME ["/app"]
+
+# install missing stuff
+RUN apk add --no-cache --repository http://dl-3.alpinelinux.org/alpine/edge/testing \
+    python make g++ git bash curl gosu && \
```
Comment (abador) [inline Dockerfile.devbox:10]: Empty continuation lines will become errors in a future release.

**Review by Grunny — COMMENTED** [review]
I can't approve my own PR, so text-only "Approved" for the new changes. ;)

File: README.md (hunk 2 above)
Comment (harnash) [reply README.md]: fix in https://github.com/Wikia/mobile-wiki/pull/1541/commits/a3a9a5db61bf464ac1573bb8df560102642ca514

**Review by abador — APPROVED** [review]

**Pull request merged**